packages/itwinui-css/src/button/default.scss

```diff
@@ -2,50 +2,39 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 @import './button';
-@import './button-icon';
 @import '../notification-marker/index';
-@import './disabled';
 
 @mixin iui-button-default {
-  background-color: var(--iui-color-background-1);
-  border-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
+  --_iui-button-text-color: var(--iui-text-color);
+  --_iui-button-background-color: var(--iui-color-background-1);
+  --_iui-button-border-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-4));
```

Comment:
But the text color uses opacity-2 and borders use opacity-4.  If `border-opacity` was a thing it would be possible but as it is right now I don't think it will work.